### PR TITLE
Add rails admin

### PR DESCRIPTION
**Pull request by eschlange**
* Added and configured devise gem.
* Added and configured rails admin gem.
* Added and configured cancancan gem.
* Added a user seed for testing.
* Added a seeding rake task.

[Finishes #95522496]

### CODE REVIEW

File: spec/models/ability_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe Ability, type: :model do
```
Comment (michaelwehrley) [inline spec/models/ability_spec.rb]: Nice!  I saw this the other day. :smile:

**Comment by michaelwehrley** [comment]
:eyeglasses: to :me!

**Comment by michaelwehrley** [comment]
@eschlange @ericcf are we good to merge despite still having `whenever` as a dependency?  Thanks!

**Comment by eschlange** [comment]
@michaelwehrley I'm fine with merging, I'm not sure when we'll get to fixing the issue in the rails_deployment project.

**Comment by ericcf** [comment]
Yes...is there a task that will need to run automatically?

**Comment by michaelwehrley** [comment]
:+1:

**Pull request merged**